mantidproject/mantid
# Dev-docs: Developer Accounts Page

Moved the "Creating a Mantid Developer Account" on the wiki to "Developer Accounts" in the new Sphinx build developer documentation, as described in #22082.

**To test:**

Build the docs and have a quick read through them to make sure everything looks correct and sounds sensible.

**NB** there is a reference to `dev-docs-git-workflow`, which does not currently link. However, this page will have that label when it is added.

*Note* you can quick build just the `.html` developer docs using the `dev-docs-html` build target.


Refs #22082. 

**Release Notes** 

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
